web/src/components/AppConfigRenderer.jsx

```diff
@@ -35,6 +35,8 @@ export class AppConfigRenderer extends React.Component {
           handleChange={this.props.handleChange || (() => { return; })}
           getData={this.props.getData || (() => { return; })}
           readonly={readonly}
+          configSequence={this.props.configSequence}
```

Comment:
Can you elaborate on what `configSequence` is and how it's used?

Reply:
configSequence is the identifier for a given config. When a customer goes to download a file we need to send the sequence number so we know which config to pull the file out of. If you look in the URL of a config page you'll see the sequence in there so we pull it out of the url and send it along through props to where the api request is made.

Reply:
gotcha thanks